controllers/provider-gcp/charts/internal/gcp-infra/templates/main.tf

```diff
@@ -121,6 +167,29 @@ output "{{ .Values.outputKeys.vpcName }}" {
   value = "{{ required "vpc.name is required" .Values.vpc.name }}"
 }
 
+{{ if .Values.create.cloudRouter -}}
+output "{{ .Values.outputKeys.cloudRouter }}" {
+  value = "${google_compute_router.router.name}"
+}
+output "{{ .Values.outputKeys.cloudNAT }}" {
+  value = "${google_compute_router_nat.nat.name}"
+}
+{{ else if and (ne .Values.vpc.cloudRouter.name "none") (ne .Values.vpc.cloudRouter.name "")}}
+output "{{ .Values.outputKeys.cloudRouter }}" {
+  value = "{{.Values.vpc.cloudRouter.name}}"
+}
+output "{{ .Values.outputKeys.cloudNAT }}" {
```

Comment:
same here - DRY

Reply:
its not duplicated, this is a different case., whats DRY?